src/engraving/libmscore/shape.cpp

```diff
@@ -121,10 +123,10 @@ double Shape::minHorizontalDistance(const Shape& a) const
             double padding = 0;
             KerningType kerningType = KerningType::NON_KERNING;
             if (item1 && item2) {
-                padding = item1->computePadding(item2);
+                padding = HorizontalSpacing::computePadding(item1, item2);
                 padding *= _squeezeFactor;
                 padding = std::max(padding, absoluteMinPadding);
-                kerningType = item1->computeKerningType(item2);
+                kerningType = HorizontalSpacing::computeKerningType(item1, item2);
```

Comment:
Better add this methods to ILayout interface and use it (layout()->computePadding(...), layout()->computeKerningType(...));
Our goal is to hide the current implementation of the layout behind the interface in order to be able to start changing it.

Reply:
All done! Let me know if I've done it right :)